api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -206,6 +205,35 @@ message ExternalProcessor {
   // Instead, the stream to the external processor will be closed. There will be no
   // more external processing for this stream from now on.
   bool disable_immediate_response = 15;
+
+  // Options related to the sending and receiving of dynamic metadata
+  MetadataOptions metadata_options = 16;
```

Comment:
Since this is related to dynamic metadata, I think the name `dynamic_metadata_options` is preferable and more clear.

Reply:
We can ignore my comment here, as dynamic in the name is redundant, since the term metadata in xDS is used specifically for dynamic metadata